flexiblepricing/views/v0/__init__.py

```diff
@@ -63,6 +62,30 @@ def get_queryset(self):
         if status_search is not None:
             queryset = queryset.filter(status=status_search)
 
+        courseware_search = self.request.query_params.get("courseware")
+        if courseware_search is not None:
```

Comment:
Can be simplified to `if courseware_search:`.

Reply:
The other 2 params name_search and status_search all have the check 'is not None', it's better to have consistent logic here with is not None IMO